document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/RelationalOperator.java

```diff
@@ -12,6 +12,7 @@ public enum RelationalOperator {
   LTE("<="),
   IN("IN"),
   CONTAINS("CONTAINS"),
+  NOT_CONTAINS("NOT_CONTAINS"),
```

Comment:
```suggestion
  NOT_CONTAINS("NOT CONTAINS"),
```
to be in sync. with "NOT IN" and "NOT EXISTS".
But, not a blocker and can come in a separate PR too.

Reply:
Done